pkg/common/common.go

```diff
@@ -0,0 +1,119 @@
+package common
+
+import (
+	"regexp"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+
+	// RoleBindingNames holds the customer rolebindings
+	RoleBindingNames = []string{
+		"customer-admin",
+		"customer-admin-project",
+		"customer-view",
+	}
+
+	rxCustomerRoleBinding = regexp.MustCompile(`(?i)^` +
+		`customer-(admin|view|admin-project)$`)
+
+	rxRestrictedNamespace = regexp.MustCompile(`(?i)^` +
```

Comment:
This is subtly wrong in multiple ways.  Maybe write it out the long way without a regexp? ;-)